src/datasource.js

```diff
@@ -933,7 +933,9 @@ ngeo.DataSource.OGCType = {
  * @enum {string}
  */
 ngeo.DataSource.WFSFeatureNS = {
-  MAPSERVER: 'http://mapserver.gis.umn.edu/mapserver'
+  GEOSERVER: 'http://www.opengis.net/gml/3.2',
```

Comment:
I don't know.  I never used GeoServer nor QGISServer.  To be sure, you should ask on mailing lists / forums.